plugin/teksi_wastewater/interlis/interlis_importer_exporter.py

```diff
@@ -554,9 +569,21 @@ def _check_subclass_count(self, schema_name, parent_name, child_list):
                         f"OK: number of subclass elements of class {parent_name} OK in schema {schema_name}!"
                     )
                 else:
-                    logger.error(
-                        f"ERROR: number of subclass elements of {parent_name} NOT CORRECT in schema {schema_name}: checksum = {parent_count} (positive number means missing entries, negative means too many subclass entries)"
-                    )
+                    if parent_count > 0:
+                        errormsg = f"Too many superclass entries for {schema_name}.{parent_name}"
+                    else:
+                        errormsg = f"Too many subclass entries for {schema_name}.{parent_name}"
+                    if limit_to_selection:
+                        logger.warning(
```

Comment:
I would add one empty line for readability :) Otherwise good for me

```suggestion
                    else:
                        errormsg = f"Too many subclass entries for {schema_name}.{parent_name}"

                    if limit_to_selection:
                        logger.warning(
```